README.md

```diff
@@ -104,6 +104,7 @@ Some major applications:
 - Emacs, editor
 - Mac Hack VI and Tech II, chess programs
 - Maclisp, interpreter and compiler
+- Muddle, interpreter
```

Comment:
I'm torn between calling it MDL or Muddle.  @36bit seems to prefer Muddle for presumably good reasons, but most manuals seem to say MDL.

Reply:
I was sticking to the terminology in the 55/105 manual: "The MDL programming language began existence in late 1970 (under the name Muddle) as a successor to Lisp ...".

I guess if we resurrect the 1973 version in the future, that would be Muddle and this is MDL.